cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java

```diff
@@ -518,7 +518,8 @@
     public static final String KAFKA_META_TABLE = "metrics.kafka.meta.table";
 
     public static final String DEFAULT_KAFKA_META_TABLE = "metrics.kafka.meta";
-    public static final String DEFAULT_KAFKA_TOPIC_PREFIX = "metrics";
+
+    public static final String DEFAULT_TOPIC_PREFIX = "metrics";
```

Comment:
Should get rid of the default in code. The default value should be in `cdap-default.xml`